Documentation/introduction/detailed_support.rst

```diff
@@ -3020,25 +3020,6 @@ RISC-V
 RISC-V Architectural Support. Basic support for the RISC-V architecture
 was contributed by Ken Pettit in NuttX-7.19.
 
-NEXT RISC-V NR5Mxx
```

Comment:
it will make the review more simple if we create a new PR for removing the old chip.

Reply:
The nr5m100 stuff was tangled in with the rv32 so it was cleaner to just remove it at the same time then have to stub stuff out for 64 and go back and remove it.